addOns/automation/src/main/java/org/zaproxy/addon/automation/VerificationData.java

```diff
@@ -231,7 +236,12 @@ public void initAuthenticationVerification(Context context, AutomationProgress p
                             this.getLoggedOutRegex()));
         }
         authMethod.setPollUrl(this.getPollUrl());
-        authMethod.setPollFrequency(JobUtils.unBox(this.getPollFrequency()));
+        if (getPollFrequency() != null) {
+            authMethod.setPollFrequency(getPollFrequency());
+        } else if (getPollFrequency() == 0) {
+            LOGGER.info("Poll Frequency should not be zero, defaulting to {}.", DEFAULT_POLL_FREQ);
+            authMethod.setPollFrequency(DEFAULT_POLL_FREQ);
```

Comment:
This is not needed, not setting allows the use of the default already. I'm not sure about the log but it should be warn.

Reply:
We should really use progress.warn instead of LOGGER

Reply:
True, we do have access to it.

Reply:
Changed